run.py

```diff
@@ -160,7 +171,8 @@ async def gateway_player(self, data: Player):
                 "ltoken": data.genshin.ltoken,
                 "cookie_token": data.genshin.cookie_token,
             }
-        china = 1 if str(uid)[0] in [1, 2, 5] else 0
+            
+        china = 1 if data.genshin.server == ServerId.CHINA else 0
```

Comment:
In gateway. It's auto detect UID is China or Oversea server
![img](https://media.discordapp.net/attachments/751301184672890945/1052506031407771668/image.png)

Reply:
that is for shenhe use